engine/api/src/org/apache/cloudstack/engine/orchestration/service/NetworkOrchestrationService.java

```diff
@@ -62,9 +63,13 @@
     static final String NetworkLockTimeoutCK = "network.lock.timeout";
     static final String GuestDomainSuffixCK = "guest.domain.suffix";
     static final String NetworkThrottlingRateCK = "network.throttling.rate";
+    static final String MinVRVersionCK = "minreq.sysvmtemplate.version";
+
+    static final ConfigKey<String> MinVRVersion = new ConfigKey<String>(String.class, MinVRVersionCK, "Advanced", "4.5.2",
```

Comment:
I guess all 4.5.x releases should support 4.5.0 as minimum VR version? so can we change this to 4.5.0?